library/psa_crypto_se.c

```diff
@@ -83,4 +123,10 @@ void psa_unregister_all_se_drivers( void )
     memset( driver_table, 0, sizeof( driver_table ) );
 }
 
+
+
+/****************************************************************/
```

Comment:
What is the reason behind this comment?

Reply:
Makes it visible that this finishes the "Driver registration" section.

Reply:
It's the end of the file anyway and nothing is after this section, that's why I ask. It seems redundant to me.

Reply:
No changes planned here.